src/utils/authentication/AuthService.js

```diff
@@ -94,4 +97,11 @@ function handleSigninCallback(dispatch, history, userManagerInstance) {
     });
 }
 
-export {userManagerPromise, login, logout, dispatchUser, handleSigninCallback, getPreLoginPath}
+function handleSilentRenewCallback(dispatch, userManagerInstance) {
+    userManagerInstance.signinSilentCallback().catch(function (e) {
+        dispatch(setLoggedUser(null));
```

Comment:
What does this do ?

Reply:
If there's an issue with the silent renew we should force our user to login explicitly, to do that we empty the user in the store.

Reply:
but this is executed in the iframe no ?

Reply:
Alright